raiden_contracts/deploy/__main__.py

```diff
@@ -311,6 +311,11 @@ def raiden(
     callback=validate_address,
     help="Address of TokenNetworkRegistry that MS contract looks at",
 )
+@click.option(
+    "--service-registry-from-deployment-file",
+    type=click.Path(exists=True),
+    help="The deployment file from which ServiceRegistry should be reused",
+)
```

Comment:
Should this default to `raiden_contracts/data/deployment_{network}.json`?

What about reusing other contracts (UDC)? It would make sense to get their addresses from the same file and not pass one file per reused contract, wouldn't it?

Reply:
The `UDC` can unfortunately not be reused because it needs to be initialized with the trusted contracts in an `init` call after deployment. This can only be done once.

The default makes sense, I'll add it.

Reply:
The default is hard to preset, because it differs based on the chain used. I added a comment in the deployment docs instead.